snmp/datadog_checks/snmp/data/conf.yaml.example

```
@@ -151,6 +151,12 @@ instances:
     #
     # enforce_mib_constraints: true
 
+    ## @param bulk_threshold - integer - optional - default: 5
+    ## The number of symbols in each tables that triggers a BULK request.
```

```suggestion
    ## The number of symbols in each table that triggers a BULK request.
```
Sorry I missed this. After this, it's 👍